### PR TITLE
Fix Dependencies for DAS when using Jetty

### LINKED ISSUE #4926: Building MOLGENIS with empty local Maven repository fails
#### Reproduce
- Create a backup of your local Maven repository
- Empty your local Maven repository
- mvn clean install 
#### Expected
- Build succes
#### Observed
- Build failure
  `[ERROR] Failed to execute goal on project molgenis-das: Could not resolve dependencies for project org.molgenis:molgenis-das:jar:1.22.0-SNAPSHOT: Could not transfer artifact javax.jms:jms:jar:1.1 from/to java.net (https://maven-repository.dev.java.net/nonav/repository): Cannot access https://maven-repository.dev.java.net/nonav/repository with type legacy using the available connector factories: BasicRepositoryConnectorFactory: Cannot access https://maven-repository.dev.java.net/nonav/repository with type legacy using the available layout factories: Maven2RepositoryLayoutFactory: Unsupported repository layout legacy -> [Help 1]`

### PR DESCRIPTION
**Pull request by Mark-de-Haan**

### CODE REVIEW

File: molgenis-app/pom.xml [hunk 1]
```diff
@@ -52,6 +52,23 @@
...
+						<version>1.6.7</version>
+						<scope>system</scope>
+						<systemPath>${basedir}/../molgenis-das/src/main/resources/mydas-1.6.7.jar</systemPath>
+					</dependency>
+					<dependency>
+						<groupId>xpp3</groupId>
+						<artifactId>xpp3</artifactId>
+						<version>1.1.3.4.O</version>
+					</dependency>
+					<dependency>
```
Comment (fdlk) [inline molgenis-app/pom.xml]: I'd expect log4j-over-slf4j instead

**Comment by dennishendriksen** [comment]
@fdlk Are you reviewing this pull request? You added comments without assigning yourself to this PR.

File: molgenis-app/pom.xml (hunk 1 above)
Comment (Mark-de-Haan) [inline molgenis-app/pom.xml]: Jetty does not run, stating it is missing the Log4j class dependency
I suspect this is caused due to MyDas being added here

Comment (dennishendriksen) [inline molgenis-app/pom.xml]: Shouldn't the dependency be excluded instead?

Comment (Mark-de-Haan) [inline molgenis-app/pom.xml]: Excluding the dependency will trigger a noClassDefFound exception

Comment (Mark-de-Haan) [inline molgenis-app/pom.xml]: The problem is, is that tomcat uses the created WAR, which contains all the correct dependencies.

Jetty does not use the war, but some fancy regex to collect all the dependencies. MyDas is not included in this dependency grab. This is why I added it here explicitly

**Comment by Mark-de-Haan** [comment]
@fdlk or @dennishendriksen ? It does not impact our own way of working seeing as I only add 3 dependencies to get Jetty running.

The WAR we deploy to servers is not affected

**Pull request merged**